### PR TITLE
Enable platform modules on MCD

### LINKED ISSUE #633: Support MCD in the platform modules
## The Problem
MCD uses a different custom field for their platform, so the original keepPlatform and multiplePlatforms modules won't work in that project.
## Possible Solution
Implement identical modules for dungeons, or somehow read both fields in the keepPlatform and multiplePlatforms modules.

### PR DESCRIPTION
**Pull request by SPGoding**
## Purpose
Fix #633.

## Approach
Read the corresponding field according to the issue's project.

#### Checklist
- [ ] Included tests
- [ ] Updated documentation in [README](https://github.com/mojira/arisa-kt/blob/master/README.md) and [wiki](https://github.com/mojira/arisa-kt/wiki)
- [ ] Tested in MCTEST-xxx

### CODE REVIEW

**Comment by chandler05** [comment]
Dungeons has other platforms that need to be added to the config at line 255

File: src/test/kotlin/io/github/mojira/arisa/modules/MultiplePlatformsModuleTest.kt [hunk 1]
```diff
@@ -238,7 +238,7 @@ class MultiplePlatformsModuleTest : StringSpec({
         val issue = mockIssue(
```
Comment (chandler05) [inline src/test/kotlin/io/github/mojira/arisa/modules/MultiplePlatformsModuleTest.kt:238]: New dungeons config values need to be added to the test module as well.

Comment (SPGoding) [reply src/test/kotlin/io/github/mojira/arisa/modules/MultiplePlatformsModuleTest.kt:238]: I was just doing that xD

File: src/main/kotlin/io/github/mojira/arisa/modules/KeepPlatformModule.kt [hunk 2]
```diff
@@ -14,20 +15,23 @@ class KeepPlatformModule(
...
         }
     }
 
-    private fun isPlatformChange(item: ChangeLogItem) =
-        item.field == "Platform"
+    private fun Issue.getPlatformValue() =
+        (if (project.key == "MCD") dungeonsPlatform else platform).getOrDefault("None")
+
+    private fun isPlatformChange(project: String, item: ChangeLogItem) =
+        item.field == (if (project == "MCD") "Platform" else "Dungeons Platform")
```
Comment (chandler05) [inline src/main/kotlin/io/github/mojira/arisa/modules/KeepPlatformModule.kt:34]:
```suggestion
        item.field == (if (project == "MCD") "Dungeons Platform" else "Platform")
```
Looks like this was backwards

Comment (SPGoding) [reply src/main/kotlin/io/github/mojira/arisa/modules/KeepPlatformModule.kt:34]: Oops, tysm!!

File: config/config.yml [hunk 3]
```diff
@@ -221,6 +222,15 @@ arisa:
         - MCPE
```
Comment (chandler05) [inline config/config.yml:222]: Looks like both modules still only run in MCPE, MCD needs to be added to the projects config here and at KeepPlatform's config.

**Pull request merged**